plugins/base/src/test/kotlin/renderers/html/DivergentTest.kt

```diff
@@ -14,9 +14,9 @@ import renderers.TestPage
 import renderers.match
 
 class DivergentTest : RenderingOnlyTestBase() {
-    private val js = SourceSetData("root", "JS", Platform.js, listOf(SourceRootImpl("pl1")))
-    private val jvm = SourceSetData("root", "JVM", Platform.jvm, listOf(SourceRootImpl("pl1")))
-    private val native = SourceSetData("root", "NATIVE", Platform.native, listOf(SourceRootImpl("pl1")))
+    private val js = SourceSetData("root", "JS", Platform.js, listOf(SourceRootImpl("pl1")), emptyList())
```

Comment:
Maybe a default in `SourceSetData` instead of explicitly writing it here? I'm not sure

Reply:
I can agree.